tasks/main.yml

```
@@ -89,52 +89,90 @@
 - name: Manage certificates
   include_tasks: certificate.yml
 
-- name: Enable SSL
-  replace:
+- name: Configure pg_hba.conf
+  become: true
+  template:
+    backup: true
+    dest: "/var/lib/pgsql/data/pg_hba.conf"
+    src: pg_hba.conf.j2
+    mode: 0600
+    owner: postgres
+    group: postgres
+  when: postgresql_pg_hba_conf is defined
+  notify: Restart postgresql
+
+- name: Create postgresql directory in /etc
+  file:
+    path: /etc/postgresql
+    state: directory
+    mode: '0755'
```

```suggestion
    mode: 0755
```
quotes or no quotes is fine, but it is better to use a consistent style